### PR TITLE
Reorganizing Compute Enums in v2018-09-30.

### PR DESCRIPTION
**Pull request by marstr**
The way that `StorageAccountTypes` is being exposed today is causing Azure/azure-cli#9133. See the thread conversation for more information.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: azure-mgmt-compute/azure/mgmt/compute/v2018_09_30/models/__init__.py [hunk 1]
```diff
@@ -84,4 +83,5 @@
     'DiskState',
     'SnapshotStorageAccountTypes',
     'AccessLevel',
+    'StorageAccountTypes',
```
Comment (lmazuel) [inline azure-mgmt-compute/azure/mgmt/compute/v2018_09_30/models/__init__.py]: Already line 79

Comment (marstr) [reply azure-mgmt-compute/azure/mgmt/compute/v2018_09_30/models/__init__.py]: Fixed in c6c8ceb

**Pull request merged**